SWADroid/src/main/res/layout/button_list.xml

```diff
@@ -0,0 +1,213 @@
+
+        <ToggleButton
+            android:id="@+id/button8"
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:layout_margin="5dp"
+            android:layout_weight="2"
+            android:background="@drawable/togglebuttonh"
+            android:textAllCaps="false"
+            android:text='h'
+            android:textOn='h'
+            android:textOff='h'
+            android:textSize="30dp"
+            android:textColor="#ffffff"
+            android:visibility="gone" />
+
+        <ToggleButton
+            android:id="@+id/button9"
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:layout_margin="5dp"
+            android:layout_weight="2"
+            android:background="@drawable/togglebuttoni"
+            android:textAllCaps="false"
+            android:text='i'
+            android:textOn='i'
+            android:textOff='i'
+            android:textSize="30dp"
+            android:textColor="#ffffff"
+            android:visibility="gone" />
+
+        <ToggleButton
+            android:id="@+id/button10"
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:layout_margin="5dp"
+            android:layout_weight="2"
+            android:background="@drawable/togglebuttonj"
+            android:textAllCaps="false"
+            android:text='j'
+            android:textOn='j'
+            android:textOff='j'
+            android:textSize="30dp"
+            android:textColor="#ffffff"
+            android:visibility="gone" />
+
+        <LinearLayout
+            android:id="@+id/linearLayout12"
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:layout_weight="1"
+            android:orientation="horizontal"
+            android:visibility="gone">
+        </LinearLayout>
+
+        <LinearLayout
+            android:id="@+id/linearLayout34"
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:layout_weight="1"
+            android:orientation="horizontal"
+            android:visibility="gone">
+        </LinearLayout>
+
+        <LinearLayout
+            android:id="@+id/linearLayout56"
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:layout_weight="1"
+            android:orientation="horizontal"
+            android:visibility="gone">
+        </LinearLayout>
+
+        <LinearLayout
+            android:id="@+id/linearLayout78"
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:layout_weight="1"
+            android:orientation="horizontal"
+            android:visibility="gone">
+        </LinearLayout>
+
+        <LinearLayout
+            android:id="@+id/linearLayout910"

```

Comment:
You should use a more descriptive name for the id. This improves the readability of the code that refers to this id and makes it easier to identify which button is being used.